Yoast/YoastSEO.js
# Updated copy and added shortlink to the proper article

## Summary

This PR can be summarized in the following changelog entry:

* Updates the copy for the `previouslyUsedKeywords` assessment by referring to a new blogpost on why it might be a bad idea to use the same keyword more than once.


## Test instructions

This PR can be tested by following these steps:

* Link this PR to your local Yoast SEO install by running `yarn link` and `yarn link yoastseo` in your Yoast SEO directory.
* Make sure to build the JS files by running `grunt build:js`.
* Create a post with a particular keyword.
* Create another post and use the same keyword as before. You shouldn't be seeing the new copy, but a different message regarding reusing keywords.
* Repeat the previous step once more and see that the `previouslyUsedKeyword` assessment now displays the new copy.
* Please verify that the link works as well.

Fixes #1334
